lib/web_ui/dev/test_platform.dart

```diff
@@ -341,22 +340,17 @@ class BrowserPlatform extends PlatformPlugin {
             value.toString() == requestData['pixelComparison']);
     final bool isCanvaskitTest = requestData['isCanvaskitTest'] as bool;
     final String result = await _diffScreenshot(
-        filename, write, maxDiffRate, region, pixelComparison, isCanvaskitTest);
+        filename, maxDiffRate, region, pixelComparison, isCanvaskitTest);
     return shelf.Response.ok(json.encode(result));
   }
 
   Future<String> _diffScreenshot(
     String filename,
-    bool write,
     double maxDiffRateFailure,
```

Comment:
Do we still use the "max diff rate" arguments for anything?

Reply:
There are several things I want to clean up in our golden tests. I'll add max diff rate to the list.